openmetrics/README.md

```diff
@@ -62,6 +62,18 @@ OpenMetrics configurations with generic wildcard values for the `metrics` option
 
 Datadog recommends that you use specific metric names or partial metric name matches for more precise collection.
 
+### Missing untyped metrics
+
+By default, the integration skips metrics that come without a type on a Prometheus exposition. If you want to collect untyped metrics, you must explicitly specify their type in the `metrics` mapping, for example:
```

Comment:
Not sure I understand what a "prometheus exposition" is. Does it get introduced in some other part of the document?

Reply:
This?
https://github.com/DataDog/integrations-core/blob/e13ee1a1aaace2a217d13a33a2c572d442ad7e12/openmetrics/README.md?plain=1#L9

Reply:
ok, all clear then!